src/Container/AbstractBusFactory.php

```diff
@@ -34,21 +40,41 @@
     abstract protected function getBusClass();
 
     /**
-     * Returns config key used within the prooph.service_bus config namespace to identify the bus.
+     * Returns the default router class to use if no one was specified in the config
      *
      * @return string
      */
-    abstract protected function getBusConfigKey();
+    abstract protected function getDefaultRouterClass();
```

Comment:
the default is really a FQCN, see https://github.com/prooph/service-bus/blob/master/src/Container/AbstractBusFactory.php#L151

It is used to be able to create a bus without further dependencies